### DISCUSSION TITLE
Not able to login to Serpbear Error - "Incorrect Username"

### DISCUSSION THREAD

**Discussion by yuvrajsagoo** [towfiqi/serpbear#74] [Q&A]
Hello,

I tried to setup Serpbear locally on Windows 11 but I'm getting "Incorrect Username". 

Next I downloaded Docker Desktop to do the same and still getting same error. 

when I open localhost:3000 I see the Add Domain page for 1 sec and it redirects to login. When I enter the login detail (from .env.local) I get "Incorrect Username" 

 Not sure what Im doing wrong. tried both Locally and Docker Desktop yet not able to login. 

let me know if you need any details.

**Comment by yuvrajsagoo** [comment]
Tried again locally on a Windows 10 and still incorrect username error locally. 


 On docker I need to add ' ' to username and password to make it work.

**Reply by gboxxx** [reply]
Getting same error, but this didn't work for me.

**Reply by towfiqi** [reply]
This issue was fixed in the latest release (v 0.2.4) . Please pull the latest release and give it a try.

**Reply by yuvrajsagoo** [reply]
Any luck on testing GSC and its pulling keywords?

**Reply by yuvrajsagoo** [reply]
Yes I confirm the authentication issue is resolved. 

Google Search Console integration is still giving 403. 

![Screenshot_10](https://user-images.githubusercontent.com/118376424/221362875-8a3d3f9a-0ddb-4dfe-bcc1-884a9d992070.jpg)
